### DISCUSSION TITLE
Cascades - ViewTemplateBody Donot Respect Hide-Body

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6777] [Ideas]
The `$:/config/ViewTemplateBodyFilters/system` in https://tiddlywiki.com/prerelease/#%24%3A%2Fconfig%2FViewTemplateBodyFilters%2Fsystem defines filters to display tiddlers using a code template.

The `hide-body` field set to `yes` forces to hide the tiddler body. But this does not work for tiddlers meet the conditions of `$:/config/ViewTemplateBodyFilters/system`.

One simple solution may be like this

```
:filter[!field:hide-body[yes]]
```

So I modified the `$:/config/ViewTemplateBodyFilters/system` and I submit a PR if this worth correcting!

**Comment by kookma** [comment]
Lets correct this:

* We asked TW to show a Tiddler body as code
* We have set `hide-body` to `yes`

Can we have both conditions at the same time? I think the current design is fine! What do you think? Should I submit a PR?

**Comment by Jermolene** [comment]
Hi @kookma perhaps all we need to do is move `$:/config/ViewTemplateBodyFilters/hide-body` further up the cascade order, so that it is evaluated before `$:/config/ViewTemplateBodyFilters/system`?

**Reply by kookma** [reply]
Make sense!
I like this solution!

**Comment by kookma** [comment]
@Jermolene the `$:/config/ViewTemplateBodyFilters/hide-body` shall be the first filter.
Now we have stylesheet filter, then system filters and number 3 is code body!
